### PR TITLE
Fix xOpera documentation project name

### PR DESCRIPTION
**Pull request by cankarm**
Fix xOpera docs title and author.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -18,9 +18,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = u'RADON'
-copyright = u'2020, RADON team'
-author = u'RADON team'
+project = u'xOpera'
+copyright = u'2020, XLAB d.o.o., xOpera team'
```
Comment (anzoman) [inline docs/conf.py:22]: I believe that the year can be updated to 2021 :smiley:

**Review by anzoman — COMMENTED** [review]
@cankarm looks good, I left just one minor comment.

File: docs/conf.py (hunk 1 above)
Comment (cankarm) [reply docs/conf.py:22]: Dammit! :D

Comment (cankarm) [reply docs/conf.py]: Here you go, @anzoman! ;)

**Comment by anzoman** [comment]
@cankarm I almost forgot -  I have one additional comment - just to be consistent with opera commits can you add a little longer and descriptive commit message (something like Fix documentation title) and then optionally you can add some commit description of you want. And also - we have two commits here now - can you squash them into one?

**Review by anzoman — APPROVED** [review]
This can be approved without addressing my comment above because the commits from this PR will be above the ones that are already applying the documentation changes.

**Pull request merged**